From: Colin Ian King <colin.king@canonical.com>
Date: Fri, 5 Jan 2018 15:50:37 +0000
Subject: scsi: qla2xxx: remove redundant assignment of d
Patch-mainline: v4.16-rc1
Git-commit: 8b56918082794e088291d3d08dac643596be5e63
References: bsc#1077338

The initialization of d is redundant as this value is never read and it
is overwritten inside the subsequent for-loop.  Remove this redundant
assignment.

Cleans up clang warning:
drivers/scsi/qla2xxx/qla_gs.c:3985:29: warning: Value stored to 'd'
during its initialization is never read

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Acked-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 drivers/scsi/qla2xxx/qla_gs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/scsi/qla2xxx/qla_gs.c
+++ b/drivers/scsi/qla2xxx/qla_gs.c
@@ -3982,7 +3982,7 @@ static void qla2x00_async_gpnft_gnnft_sp
 		(struct ct_sns_req *)sp->u.iocb_cmd.u.ctarg.req;
 	struct ct_sns_gpnft_rsp *ct_rsp =
 		(struct ct_sns_gpnft_rsp *)sp->u.iocb_cmd.u.ctarg.rsp;
-	struct ct_sns_gpn_ft_data *d = &ct_rsp->entries[0];
+	struct ct_sns_gpn_ft_data *d;
 	struct fab_scan_rp *rp;
 	int i, j, k;
 	u16 cmd = be16_to_cpu(ct_req->command);
